packages/ic-sha3/README.md

```diff
@@ -0,0 +1,3 @@
+# SHA-3
+
+Utilities for hashing data with algorithms from the SHA-3 standard (e.g., Keccak256, Shake256, etc.).
```

Comment:
Optionally we could describe that this is basically a wrapper of `sha3` for the Internet Computer ecosystem, and what the added value of using `ic-sha3` vs `sha3` is.

Reply:
makes sense. I reformulated it.